ExtractorLauncher/config/config.yml

```
@@ -265,6 +265,21 @@ logger:
         level: "information"
         path: "logs/log.txt"
 
+
+metrics:
+    # otherMetrics:
+    #   - namespaceUri: "KEPServerEX"
+
+influx-db-metrics:
+     enabled: true
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 5

(indentation)